payments/models.py

```diff
@@ -444,6 +472,53 @@ def __str__(self):
         )
 
 
+class SAPIncomeAccount(models.Model):
```

Comment:
It would probably be good idea to have class-level comment on what `SAPIncomeAccount` and `SAPMaterialCode` are as it might not be clear to maintainers.

Reply:
Added comments.

Reply:
I'll add documentation for the whole invoicing feature separately.